libs/@guardian/atoms-rendering/package.json

```diff
@@ -2,7 +2,7 @@
 	"name": "@guardian/atoms-rendering",
 	"version": "26.0.0",
 	"dependencies": {
-		"is-mobile": "^3.1.1"
```

Comment:
Bad bot! Direct dependencies should be allowed a range.

Reply:
In its description it has the line:

> Add the preset :preserveSemverRanges to your config if you don't want to pin your dependencies.

Is that something we could add, or would that mess things up? 🤔

Reply:
Not sure what the docs say, but I think we want pinned `devDependencies` and ranges for direct `dependencies` if that's a possibility.

Reply:
why?

Reply:
Thanks for the question @sndrs. It made me realise that pinning is probably better.

I originally thought allowing ranges in direct dependencies may help reduce the number of distinct dependencies consumers will need. But then realised if you think consumers will already have the dependency locally, maybe you should make it a peer?

So: Pinning everything but peer dependencies is safer and the only way you can guarantee your package will work. There's a reason Renovate recommends it. Let's keep it that way!